File: android/app/src/main/java/info/bagen/dwebbrowser/microService/user/ToyJMM.kt
package info.bagen.dwebbrowser.microService.user

import info.bagen.dwebbrowser.microService.browser.jmm.JmmMetadata
import info.bagen.dwebbrowser.microService.browser.jmm.JsMicroModule

class ToyJMM : JsMicroModule(
    JmmMetadata(
        id = "toy.bfs.dweb",
        server = JmmMetadata.MainServer(
            root = "/sys",
            entry = "public.service.worker.js"
        ),
    )
) {}

File: next/dweb-browser/src/Platforms/iOS/MicroModule/User/ToyJMM.cs
using DwebBrowser.MicroService.Browser.Jmm;
namespace DwebBrowser.MicroService.User;

public class ToyJMM : JsMicroModule
{
    public ToyJMM() : base(new JmmMetadata(
        "toy.bfs.dweb",
        new JmmMetadata.MainServer() { Root = "file:///jmm", Entry = "/toy.worker.js" }))
    {
    }
}

